Wikipedia/Code/WMFPasswordResetter.swift

```diff
@@ -2,14 +2,25 @@
 public enum WMFPasswordResetterError: LocalizedError {
     case cannotExtractResetStatus
     case resetStatusNotSuccess
+    case accountError(code: String, message: String)
+
     public var errorDescription: String? {
         switch self {
         case .cannotExtractResetStatus:
             return "Could not extract status"
         case .resetStatusNotSuccess:
             return "Password reset did not succeed"
+        case .accountError(let code, let message) where WMFPasswordResetterError.supportedAccountErrorCodes.contains(code):
+            return message
+        default:
+            return CommonStrings.genericErrorDescription
         }
     }
+
+    private static let supportedAccountErrorCodes = [
+        "ratelimited",
+        "noemail"
```

Comment:
should we be selective here or allow all errors to be shown? I thought html could creep in like here `https://phabricator.wikimedia.org/T217186` but not sure if that would ever be the case

Reply:
Even if it has html/wikitext it's prob good to show... hopefully these continue to be improved upstream...

Reply:
I think we can allow all errors to be shown if they're parsed properly - It looks like it would need to be parsed initially with `action=parse` then we could convert to an `NSAttributedString` from HTML with links

Reply:
Oh... so if the error string has wikitext hit the api to convert to html?

Reply:
Yeah - but that could be handled separate from this PR. I think for this PR just pass whatever error message MW is sending back, then the fix for https://phabricator.wikimedia.org/T217186  will handle the issue of wikitext and html in errors